src/LedgerArk.js

```diff
@@ -34,18 +34,18 @@ LedgerArk.prototype.getAddress_async = function(path) {
 
 LedgerArk.prototype.signTransaction_async = function(path, rawTxHex) {
 	var splitPath = utils.splitPath(path);
-	var rawTx = new Buffer(rawTxHex, 'hex');
+	var rawTx = Buffer.from(rawTxHex, 'hex');
 	var self = this;
 
 	var data1, data2;
-	var data1_headerlength = new Buffer(2);
-	var data2_headerlength = new Buffer(1);
+	var data1_headerlength = Buffer.from(2);
+	var data2_headerlength = Buffer.from(1);
```

Comment:
If you wish to create a new Buffer of a specific length you need to use Buffer.alloc